printer_mods/jeoje/4.3_Inch_Touchscreen_Mount/README.md

```diff
@@ -1,18 +1,19 @@
 # Voron v2.4 4.3" Touchscreen Mount
 
-Inspired by [Roboticator's excellent 4" touchscreen mount](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/roboticator24/4inch_touchscreen_mount_for_v2.4), I wanted to use a capacitive touchscreen and a DSI connection instead.
+Inspired by [Roboticator's excellent 4" touchscreen mount](../../roboticator24/4inch_touchscreen_mount_for_v2.4), I wanted to use a capacitive touchscreen and a DSI connection instead.
 
 This mod mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) purchased from [Amazon](https://smile.amazon.com/gp/product/B083TG7Y9B/).  The screen is oriented at a 60 degree angle keeping the height the same as a standard skirt.
 
 <div align="center">
     <img src="images/CAD_front_profile.png" height="400">
     <img src="images/CAD_side_profile.png" height="400">
+    <img src="images/CAD_offset_profile.png" height="400">
     <img src="images/CAD_mount_side_profile_angle.png" height="400">
     <img src="images/CAD_mount_offset_profile.png" height="400">
 </div>
 
 
-#### Combine with [Roboticator's modified skirts](https://github.com/roboticator24/VoronUsers/tree/master/printer_mods/roboticator24/front%2Brear_skirts_for_v2.4) for a cleaner look.
+#### Combine with [Roboticator's modified skirts](../../roboticator24/front%2Brear_skirts_for_v2.4) for a cleaner look.
```

Comment:
This hasn't been merged yet, so it's a dead link if you make it relative.  Can you change it back to

    [Roboticator's modified skirts](https://github.com/roboticator24/VoronUsers/tree/master/printer_mods/roboticator24/front%2Brear_skirts_for_v2.4)